src/verifiers/issuerIdentity/dnsTxt/openAttestationDnsTxt.ts

```diff
@@ -156,42 +165,47 @@ const verifyV3 = async (document: v3.WrappedDocument, options: VerifierOptions):
   const { identifier } = document.openAttestationMetadata.identityProof;
   const issuerIdentity = await resolveIssuerIdentity(identifier, smartContractAddress, options);
 
+  if (InvalidDnsTxtVerificationStatus.guard(issuerIdentity)) {
+    return {
+      name,
+      type,
+      data: {
+        identifier: issuerIdentity.location,
+        value: issuerIdentity.value,
+      },
+      reason: issuerIdentity.reason,
+      status: "INVALID",
+    };
+  }
   return {
     name,
     type,
     data: {
       identifier: issuerIdentity.location,
       value: issuerIdentity.value,
     },
-    ...(issuerIdentity.status === "INVALID" ? { reason: issuerIdentity.reason } : {}),
     status: issuerIdentity.status,
```

Comment:
This is awkward now, when I highlight, ts tells me its "VALID", so might as well put it as valid?

Reply:
Also, i think its better to guard the positive case and return invalid on else in case the guard fails to catch something?

Reply:
Can change to "VALID", I don't care much, same for the order

Reply:
done